templates/cluster.yml.j2

```diff
@@ -7,6 +7,12 @@ spec:
   api:
     loadBalancer:
       type: Public
+{% if cluster.api_additional_sgs is defined and cluster.api_additional_sgs|length > 0 %}
+      additionalSecurityGroups:
```

Comment:
Please also add this as an example to the README.md that people who never worked with this role know what can be done.

Reply:
@cytopia done